packages/scripts/src/upgrade-tests/voting2/2_Verify.ts

```diff
@@ -86,15 +98,54 @@ async function main() {
   console.log("✅ Voting has been successfully migrated!");
 
   console.log(" 4. Validating old voting contract and finder is owned by governor v2...");
-  assert.equal(await oldVoting.owner(), governorV2Address);
-  assert.equal(await finder.owner(), governorV2Address);
+  assert.equal(await oldVoting.owner(), governorV2.address);
+  assert.equal(await finder.owner(), governorV2.address);
   console.log("✅ Old Voting & finder correctly transferred ownership back to governor v2!");
 
   console.log(" 5. Governor v2 is the owner of the voting token...");
-  assert.equal((await votingToken.getMember(0)).toLowerCase(), governorV2Address.toLowerCase());
+  assert.equal((await votingToken.getMember(0)).toLowerCase(), governorV2.address.toLowerCase());
   console.log("✅ Voting token owner role correctly set!");
 
-  console.log("Verified!");
+  console.log(" 6. Governor v2 is the owner of proposer...");
+  assert.equal((await proposer.owner()).toLowerCase(), governorV2.address.toLowerCase());
+  console.log("✅ Proposer owner role correctly set!");
+
+  console.log(" 7. Governor v2 is the owner of proposer v2...");
+  assert.equal((await proposerV2.owner()).toLowerCase(), governorV2.address.toLowerCase());
+  console.log("✅ Proposer v2 owner role correctly set!");
+
+  console.log(" 8. Governor v2 is registered in the registry...");
+  assert(await registry.isContractRegistered(governorV2.address));
+  console.log("✅ Governor v2 registered in registry!");
+
+  console.log(" 9. Proposer v2 is registered in the regstry...");
+  assert(await registry.isContractRegistered(proposerV2.address));
+  console.log("✅ Proposer v2 registered in registry!");
+
+  console.log(" 10. Governor v2 received all the voting tokens from Governor...");
+  assert((await votingToken.balanceOf(governorV2.address)).gt(hre.web3.utils.toWei("30000000", "ether")));
```

Comment:
Maybe this hardcoded value is unnecessary and we should just check that the balance is >0